echopype/tests/test_convert.py

```diff
@@ -335,23 +331,30 @@ def test_convert_ek60(
     if output_save_path and output_save_path.startswith("s3://"):
         output_storage_options = common_storage_options
 
-    ec = open_raw(file=input_path, model=model, storage_options=input_storage_options)
+    # Only using one file
+    ec = open_raw(raw_file=ipath, sonar_model=model, storage_options=input_storage_options)
 
     if (
         export_engine == "netcdf4"
         and output_save_path is not None
         and output_save_path.startswith("s3://")
     ):
         return
-    ec._to_file(
-        engine=export_engine,
+
+    if export_engine == "netcdf4":
+        to_file = getattr(ec, "to_netcdf")
+    elif export_engine == "zarr":
+        to_file = getattr(ec, "to_zarr")
+    else:
+        return
+
+    to_file(
         save_path=output_save_path,
         overwrite=True,
-        combine=combine_files,
         storage_options=output_storage_options,
```

Comment:
The attribute should be `output_storage_options`. https://github.com/ngkavin/echopype/blob/86895a8134930f8cd6370c1b441aaa850ee1b873/echopype/convert/api.py#L122

Reply:
cc @ngkavin you can either change it on api.py or the test file. This is why it's failing because the correct credentials is not actually being used.